fastserde/avro-fastserde-tests110/build/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_10/FastSerdeLogicalTypesDefined_GenericDeserializer_229156053_229156053.java

```diff
@@ -14,13 +19,29 @@ public class FastSerdeLogicalTypesDefined_GenericDeserializer_229156053_22915605
 {
 
     private final Schema readerSchema;
+    private final GenericData modelData;
+    private final org.apache.avro.data.TimeConversions.DateConversion conversion_date = new org.apache.avro.data.TimeConversions.DateConversion();
+    private final org.apache.avro.data.TimeConversions.TimeMillisConversion conversion_time_millis = new org.apache.avro.data.TimeConversions.TimeMillisConversion();
+    private final org.apache.avro.data.TimeConversions.TimestampMillisConversion conversion_timestamp_millis = new org.apache.avro.data.TimeConversions.TimestampMillisConversion();
+    private final Schema timeMillisField0;
+    private final Schema logicalTypeSchema__419105534 = Schema.parse("{\"type\":\"int\",\"logicalType\":\"time-millis\"}");
+    private final Schema dateField0;
+    private final Schema logicalTypeSchema__59052268 = Schema.parse("{\"type\":\"int\",\"logicalType\":\"date\"}");
     private final Schema arrayOfUnionOfDateAndTimestampMillis0;
     private final Schema arrayOfUnionOfDateAndTimestampMillisArrayElemSchema0;
+    private final Schema arrayOfUnionOfDateAndTimestampMillisElemOptionSchema0;
+    private final Schema arrayOfUnionOfDateAndTimestampMillisElemOptionSchema1;
+    private final Schema logicalTypeSchema_1074306973 = Schema.parse("{\"type\":\"long\",\"logicalType\":\"timestamp-millis\"}");
 
-    public FastSerdeLogicalTypesDefined_GenericDeserializer_229156053_229156053(Schema readerSchema) {
+    public FastSerdeLogicalTypesDefined_GenericDeserializer_229156053_229156053(Schema readerSchema, GenericData modelData) {
         this.readerSchema = readerSchema;
+        this.modelData = modelData;
```

Comment:
Do we need to pass in the `modelData` and hang on to it as an instance property? I don't see any use of it in the generated code... I could have missed it maybe?

Reply:
Thanks! You seems to be right. I will revisit the code and remove if it unless it's somewhere needed.
It's probably leftover from an early implementation.

Reply:
You were right! Thanks again!
I pushed the fixes in 9th commit.
To avoid force-pushing whole branch (GH may end up with dangling comments) I re-created the commits with fixes:
- https://github.com/RTBHOUSE/avro-util/pull/8 (temp PR, just to see the results in generated classes)
Files in "AFTER" commits now have no modelData field.